global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I'm not sure we'd want this in our repo root. 

It makes d2d development hard and cause build failures without further changes.

Reply:
Would an empty global.json be acceptable?  From [the docs](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json) it should have no effect:
> If no global.json file is found, or global.json doesn't specify an SDK version nor an allowPrerelease value, the highest installed SDK version is used (equivalent to setting rollForward to latestMajor).

Reply:
I guess that'd be fine.

What's driving the requirement for a global.json?

Reply:
Arcade looks for this in `RepoRoot`.  It wants to install the SDK specified in the global.json, but this approach bypasses that part - unfortunately the search for global.json happens much sooner.  It also expects the `.git` directory to be in the same directory, so we can't cheat it by telling it that `RepoRoot` is actually `eng/source-build`.  I replaced the contents of global.json with a comment that I hope explains this all.